packages/dcos-net/extra/dcos-net-setup.py

```diff
@@ -30,38 +31,53 @@ def main():
         sys.stderr.buffer.write(result.stderr)
         if result.stderr.strip().endswith(b'File exists'):
             result.returncode = 0
-    elif sys.argv[1] == 'iptables':
+        sys.exit(result.returncode)
```

Comment:
The reason why the earlier code was consuming the errors was that it was harmless. I think with this new code you will have issues with the restart of `dcos-net`

Reply:
The issues can also be consumed because we remained `result.returncode = 0` before `sys.exit(result.returncode)`?